rovilay/phone_number-generator
# Setup server

#### What does this PR do?
This PR covers the tasks outlined below 👇🏽

#### Description of Task to be completed?
    * setup routes
    * add error handler middleware
    * setup cors

#### How should this be manually tested?
- git clone this app - `git clone https://github.com/rovilay/phone_number-generator.git`
- navigate to the cloned app
- git checkout to this branch - `git checkout -b ch-setup-server`
- set up the `.env` with the `.env.sample` file
- install dependencies - `yarn install`
- start the app - `yarn run server`
- navigate to `localhost:port` on the browser
- you should see a welcome message - `Welcome to phone_number generator app`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
